axelrod/tests/unit/test_graph.py

```diff
@@ -86,6 +86,87 @@ def test_init(self):
             self.assertEqual(g.in_mapping[node], expected_in_mapping[node])
         self.assertEqual(g._edges, expected_edges)
 
+    def test_out_dict(self):
+        # Undirected graph with vertices and unweighted edges
+        g = graph.Graph(edges=[[1, 2], [2, 3]])
+        expected_out_mapping = {
+            1: {2: None},
+            2: {1: None, 3: None},
+            3: {2: None}
+        }
+        for key in expected_out_mapping:
```

Comment:
`assertDictEqual` is better since this loop just tests that one dict is a subset of the other.